README.md

```diff
@@ -1,92 +1,111 @@
-# SAS Viya 4 Infrastructure as Code (IaC) for Microsoft Azure
+# SAS Viya Infrastructure as Code (IaC) for Microsoft Azure
 
 ## Overview
 
-This project contains Terraform scripts to provision Microsoft Azure Cloud infrastructure resources required to deploy SAS Viya 4 products. Here is a list of resources this project can create -
+This project contains Terraform scripts to provision the Microsoft Azure Cloud infrastructure resources that are required to deploy SAS Viya product offerings. Here is a list of resources that this project can create:
 
-  >- Azure Resource Group(s): Primary Resource Group and AKS Resource Group
-  >- Virtual Network, Network Security Groups and Network Security Rules
+  >- Azure resource group(s): primary resource group and AKS resource group
+  >- Virtual network, network security groups, and network security rules
   >- Managed Azure Kubernetes Service (AKS) cluster
-  >- System and User AKS Node pools with required Labels and Taints
-  >- Infrastructure to deploy SAS Viya CAS in SMP or MPP mode
-  >- Storage options for SAS Viya -  AzureDisk/Files(dev) or NFS Server or Azure NetApp Files(HA)
-  >- Azure DB for PostgreSQL, optional
-  >- Azure Container Registry, optional
+  >- System and user AKS node pools with required labels and taints
+  >- Infrastructure to deploy the SAS Viya CAS server in SMP or MPP mode
+  >- Storage options for SAS Viya:  Azure Premium Managed Disks (standard deployments); NFS Server or Azure NetApp Files (HA deployments)
+  >- Azure Database for PostgreSQL (optional)
+  >- Azure Container Registry (optional)
```

Comment:
So here, the wording and capitalization is being used to identify items relevant to the provider. Is there a reason the casing was adjusted? It should be reverted.

This needs to be removed: _Azure Premium Managed Disks (standard deployments);_ 

_NFS Server_ should be NFS Server (standard)
_Azure NetApp Files (HA deployments)_ should just have (HA) 

These values are part of the code base and identify with a given storage type.